shell/renderer/api/electron_api_context_bridge.cc

```diff
@@ -607,6 +607,59 @@ void ExposeAPIInMainWorld(v8::Isolate* isolate,
   }
 }
 
+void ExposeAPIInIsolatedWorld(v8::Isolate* isolate,
+                              const int world_id,
+                              const std::string& key,
+                              v8::Local<v8::Value> api,
+                              gin_helper::Arguments* args) {
+  TRACE_EVENT1("electron", "ContextBridge::ExposeAPIInIsolatedWorld", "worldId",
+               world_id);
```

Comment:
For consistency with `ExposeAPIInMainWorld`, let's continue to use the `key` argument.
```suggestion
  TRACE_EVENT2("electron", "ContextBridge::ExposeAPIInIsolatedWorld", "key", key, "worldId",
               world_id);
```